File: Models/Transfer.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using Transfers.Models;

namespace Transfers {

    public class Transfer {
        public int Id { get; set; }
        public DateTime DateIn { get; set; }
        public int Adults { get; set; }
        public int kids { get; set; }
        public int Free { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
        public int TotalPersons { get; set; }
        public string Remarks { get; set; }
        public string UserName { get; set; }
        public int CustomerId { get; set; }
        public int PickupPointId { get; set; }
        public int DestinationId { get; set; }
        public int DriverId { get; set; }
        public int PortId { get; set; }
        public Customer Customer { get; set; }
        public Destination Destination { get; set; }
        public PickupPoint PickupPoint { get; set; }
        public Driver Driver { get; set; }
    }

}

File: ClientApp/src/app/models/transfer.ts
export interface ITransfer {
    id: number;
    dateIn: string;
    customer: {
        id: number
        description: string
    },
    pickupPoint: {
        id: number,
        description: string
        route: {
            id: number
            description: string
        }
    }
    destination: {
        id: number
        shortDescription: string
        description: string
    }
    adults: number;
    kids: number;
    free: number;
    totalPersons: number;
    remarks: string;
    userName: string
}
